src/components/container/ProviderSearch/ProviderSearch.stories.tsx

```diff
@@ -36,8 +36,19 @@ onProviderSelected.parameters = {
 
 export const Live = Template.bind({});
 Live.args = {
-	providerCategories: ["Health Plan"]
+	//providerCategories: ["Health Plan"]
 }
 Live.parameters = {
 	externalAccounts: []
+}
+
+export const LiveStandalone = Template.bind({});
+LiveStandalone.args = {
+	connectExternalAccountOptions: {
+		openNewWindow: false,
+		standaloneModeFinalRedirectPath: "https://mydatahelps.org"// replace with actual redirect url for this to work
```

Comment:
Should this be a todo?

Reply:
I think we don't have an actual published / live redirect url we have shared here, so not really a TODO, more if you are doing local dev here is where you would put in your redirect url